<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="236" started-at="2020-05-06T22:13:57 IST" finished-at="2020-05-06T22:13:57 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="236" started-at="2020-05-06T22:13:57 IST" finished-at="2020-05-06T22:13:57 IST">
      <class name="TestNG_Listners.TestNG_Listner_Demo">
        <test-method status="PASS" signature="test1()[pri:0, instance:TestNG_Listners.TestNG_Listner_Demo@3224f60b]" name="test1" duration-ms="65" started-at="2020-05-06T22:13:57 IST" finished-at="2020-05-06T22:13:57 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="test2()[pri:0, instance:TestNG_Listners.TestNG_Listner_Demo@3224f60b]" name="test2" duration-ms="10" started-at="2020-05-06T22:13:57 IST" finished-at="2020-05-06T22:13:57 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method status="SKIP" signature="test3()[pri:0, instance:TestNG_Listners.TestNG_Listner_Demo@3224f60b]" name="test3" duration-ms="7" started-at="2020-05-06T22:13:57 IST" finished-at="2020-05-06T22:13:57 IST">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[This test is skipped]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: This test is skipped
at TestNG_Listners.TestNG_Listner_Demo.test3(TestNG_Listner_Demo.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
      </class> <!-- TestNG_Listners.TestNG_Listner_Demo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
